mappings/net/minecraft/item/FoodItemSetting.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS awp net/minecraft/item/FoodItemSetting
+	CLASS awp$a Builder
+		FIELD a hunger I
+		FIELD b saturation F
+		FIELD c wolfFood Z
+		FIELD d alwaysEdible Z
+		FIELD e eatenFast Z
+		FIELD f statusEffectChances Ljava/util/List;
+		METHOD a wolfFood ()Lawp$a;
+		METHOD a setSaturation (F)Lawp$a;
+			ARG 1 amount
+		METHOD a setHunger (I)Lawp$a;
+			ARG 1 amount
+		METHOD a addStatusEffect (Lahq;F)Lawp$a;
+			ARG 2 chance
+		METHOD b alwaysEdible ()Lawp$a;
+		METHOD c eatenFast ()Lawp$a;
+		METHOD d build ()Lawp;
+	FIELD a hunger I
+	FIELD b saturation F
+	FIELD c wolfFood Z
+	FIELD d alwaysEdible Z
+	FIELD e eatenFast Z
+	FIELD f statusEffectsChances Ljava/util/List;
```

Comment:
statusEffectChances, with singular "effect"

Reply:
Its not a single effect, its a `List<Pair<StatusEffectInstance, Float>>`